config/routes.rb

```diff
@@ -10,6 +10,8 @@
   # first created -> highest priority.
 
   get "/robots.txt" => RobotsGenerator
+  get "/sitemap.xml.gz" => "sitemap#sitemap", format: :xml
```

Comment:
> Speaking of that, I also need to add an entry to the robots.txt file with the sitemap name. Will do that.

Actually, looks like the robots.txt file isn't generated dynamically, so we can't point to the sitemap.xml.gz file there (and the link in there needs to be absolute, can't be relative: https://moz.com/community/q/robots-txt-sitemap-with-relative-path )